src/components/CookieClicker.vue

```diff
@@ -0,0 +1,411 @@
+<template>
+    <div class="cookie-clicker">
+      <div class="cookie-container">
+        <h1>Cookies: {{ formattedCookies }}</h1>
+        <h2>CPS: {{ cookiesPerSecond.toFixed(1) }}</h2>
+        <button class="cookie" @click="increment">
+          <img src="../assets/cookie.png" alt="Cookie" @error="handleCookieError"/>
```

Comment:
You can import images with es so you don't have to do this. not sure if this even works properly.